server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java

```diff
@@ -321,6 +321,13 @@ public ChangeScheduleResp changeSchedule(ScheduleChangeParams req) {
                 log.warn("Delete schedule is not allowed,status={}", targetSchedule.getStatus());
                 throw new IllegalStateException(
                         "Delete schedule is not allowed, only can delete terminated schedule or finished schedule.");
+            } else {
+                List<ScheduleTaskEntity> runningTaskList = scheduleTaskRepository.findByJobNameAndStatusIn(
```

Comment:
Please use the scheduleService.hasExecutingTask method to resolve this issue

Reply:
Already changed.